### ISSUE TITLE
Had to explictly install d3

### ISSUE THREAD

**Issue by apoorv-kumar** [cbgaindia/story-generator#78]
For some reason npm install didn't automatically install d3 and I was getting the following error - 

> ERROR in ./~/react-vis/dist/plot/axis/decorative-axis.js
> Module not found: Error: Cannot resolve module 'd3-format' in /home/apoorv/sandbox/dkblr/story-generator/node_modules/react-vis/dist/plot/axis
>  @ ./~/react-vis/dist/plot/axis/decorative-axis.js 15:16-36
> 
> ERROR in ./~/react-vis/dist/radar-chart/index.js
> Module not found: Error: Cannot resolve module 'd3-format' in /home/apoorv/sandbox/dkblr/story-generator/node_modules/react-vis/dist/radar-chart
>  @ ./~/react-vis/dist/radar-chart/index.js 39:16-36
> 

I did _npm install d3_ and then it _npm start -s_ works.

**Comment by tejasrivastav** [comment]
@apoorv-kumar d3 isnt added as dependency in the `package.json`.

**Comment by apoorv-kumar** [comment]
@tejasrivastav It should be added right ? The project doesn't seem to work otherwise.

**Comment by akshay2905** [comment]
@tejasrivastav @apoorv-kumar True, D3 isnt added as dependency in the package.json but has been added explicitly to support topojson to convert topojson data to geojson. I was having issues in installation and importing of the modules, so I included in explicitly. There arn't any other dependency for d3. The visualization are built using react-vis and vanilla js. React-vis uses d3 built for that it doesn't need to be included additionally.

**Comment by tejasrivastav** [comment]
@akshay2905 d3 isnt added as dep in react-viz [repo](https://github.com/uber/react-vis/blob/master/package.json) correct if im referring to same

**Comment by akshay2905** [comment]
@tejasrivastav it is actually. It is based on React and d3. Sharing the package.json file for react-vis - [Link](https://github.com/uber/react-vis/blob/98841657bfacf5b03bb0070d5104e807fa5dddcf/package.json)

They have included invidual components of the d3 library to serve different purposes

**Comment by apoorv-kumar** [comment]
@akshay2905 I see. It might be useful to include this additional step in the installation documentation. I'm new to the scene so it wasn't obvious to me.

**Comment by akshay2905** [comment]
@apoorv-kumar We might need to probably diagnose the issue here. As it is not a reported as common issue by others. Managing packages with npm can be a little tricky at times. There might be other issue at play. At what stage did you face this problem?

**Comment by apoorv-kumar** [comment]
During my first _npm start -s_, I got the error I shared above. It might be that typically people have d3 installed in the global npm libs and I didn't.

**Comment by akshay2905** [comment]
@apoorv-kumar  Possible. Will check with a few others if the same problem exist.
